packages/webdriver-utils/src/providers/automateProvider.js

```diff
@@ -133,18 +133,19 @@ export default class AutomateProvider extends GenericProvider {
     const tiles = [];
     const tileResponse = JSON.parse(responseValue.result);
     log.debug('Tiles captured successfully');
-
+    const windowHeight = (await this.driver.executeScript({ script: 'return window.innerHeight;', args: [] })).value;
+    const dpr = (await this.driver.executeScript({ script: 'return window.devicePixelRatio;', args: [] })).value;
```

Comment:
Please try and move this to `percyScreenshotBegin` in seleniumHub

Reply:
We can only do dpr in `percyScreenshotBegin` as windowHeight changes for every tile.